src/operator.rs

```diff
@@ -109,6 +109,7 @@ where
     type Param = ParamKind;
     type Output = f64;
     type Hessian = ();
+    type Jacobian = ParamKind;
```

Comment:
I had to add this to compile, as otherwise I was getting,
```
   Compiling pyargmin v0.0.0 (/home/rth/src/pyargmin)
error[E0046]: not all trait items implemented, missing: `Jacobian`
   --> src/operator.rs:105:1
    |
105 | / impl ArgminOp for PyArgminOp
106 | | where
107 | |     PyArgminOp: Clone,
108 | | {
...   |
156 | |     }
157 | | }
    | |_^ missing `Jacobian` in implementation
    |
    = note: `Jacobian` from trait: `type Jacobian;`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0046`.
```
even before refactoring in this PR. I wonder if it could be related to https://github.com/argmin-rs/pyargmin/issues/1

Reply:
This is because I made changes to `argmin` but forgot to update `pyargmin` accordingly.